Sources/SourceKitLSP/Swift/Refactoring.swift

```diff
@@ -0,0 +1,124 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import LanguageServerProtocol
+import SourceKitD
+
+protocol Refactoring {
```

Comment:
Would `RefactoringResult` or `RefactoringResponse` make more sense as the name here? `Refactoring` sounds more like what you implemented as `RefactorCommand` to me.